partcad/src/partcad/wrappers/wrapper_openscad.py

```
@@ -0,0 +1,80 @@
+        # Handle strings, numbers, or any other type by converting to string
+        args_str = str(build_parameters)
+
+    new_line = f"{method}({args_str});"
+
+    with open(tmp_file_path, "r") as f:
+        lines = f.readlines()
+
+    base_dir = os.path.dirname(path)
+    for line in lines:
+        line_stripped = line.strip()
+        if line_stripped.startswith("use <") or line_stripped.startswith("include <"):
```

Ruff (0.8.2)

Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)